common/app/views/support/GoogleAnalyticsAccount.scala

```diff
@@ -4,7 +4,8 @@ import conf.Configuration.environment
 
 object GoogleAnalyticsAccount {
 
-  case class Tracker(trackingId: String, trackerName: String, samplePercentage: Int = 100)
+  // NOTE that the 'samples rates' when set to 0, seem to be 100%
+  case class Tracker(trackingId: String, trackerName: String, samplePercentage: Int = 100, siteSpeedSamplePercentage: Double = 0.1)
```

Comment:
I actually do want it to be 0.1% sample for these (equals about 17k pageviews a day, enough for these metrics I think)